### PR TITLE
🧹 Clean up styling and misc things

### PR DESCRIPTION
**Pull request by Qrtn**
## Status:

:construction: In development

## Description

Some final cleanup for MVP.

Fixes #<number>

### Jeffrey
- [x] Fix Entry Back Button (Replace history w/ correct results page rather than going back)
- [x] Box shadow on resizer
- [x] Clicking on a cluster or location sometimes brings up the Google Maps info box (click event should not be passed through)
- [x] Don't show resizable cursor when pane is not resizable
- [x] Add opacity in the location icons
- [x] EatTheWeeds svg to png
- [x] Fix scrolling on entry details page on mobile
- [x] Add key prop to EntryDetails list
- [x] Remove extra styled components in EntryDetails

### Siraj
- [x] New photos display for desktop entries
- [x] Fix alt text
- [x] Desktop navbar mockup
- [x] Resolve all mentioned TODOs
- [x] Add border-radius (animated from search to none for entries)

### CODE REVIEW

**Comment by code-arman** [comment]
r we gonna modify the design of the blue circle icon on the locations before mvp?

**Comment by Qrtn** [comment]
Reopening -- closed in order to fix up this PR's commit history.

**Comment by Qrtn** [comment]
> r we gonna modify the design of the blue circle icon on the locations before mvp?

Hmm yes I think some opacity is good. We'll add that.

0.6:
image

0.7:
image

0.75:
image

File: src/components/entry/EntryDetails.js [hunk 1]
```diff
@@ -147,10 +247,37 @@ const EntryDetails = ({ isDesktop }) => {
   return locationData && locationTypeData ? (
     <EntryDetailsPageContainer isDesktop={isDesktop}>
       {locationData.photos.length > 0 && (
-        <ImageContainer
-          src={locationData.photos[0].photo.original}
-          alt="entry-details-photo"
-        />
+        // TODO: use alt based off of photo description or filename
+        <PhotoGrid>
```
Comment (Qrtn) [inline src/components/entry/EntryDetails.js:251]: TODO: extract out photo grid into its own component

**Review by Archna-1 — APPROVED** [review]
LGTM! :)

**Review by code-arman — APPROVED** [review]
LGTM

**Pull request merged**